ProductsWebApi/ProductsWebApi.Tests/ProductsServiceTest.cs

```diff
@@ -0,0 +1,123 @@
+﻿using AutoFixture;
+using FluentAssertions;
+using Moq;
+using ProductsWebApi.Interface;
+using ProductsWebApi.Models;
+using ProductsWebApi.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ProductWebApi.Tests
+{
+    public class ProductsServiceTest
+    {
+        private readonly Mock<IProductsService> _mockproductService;
+        private readonly ProductsService _productService;
+        private readonly Fixture _fixture;
+        private string expectedNullResult = null;
+        private Product productRequestObj = null;
+
+        public ProductsServiceTest()
+        {
+            _mockproductService = new Mock<IProductsService>();
+            _productService = new ProductsService(_mockproductService.Object);
+            _fixture = new Fixture();
+        }
+
+        [Fact]
+        public async Task GetProduct_ValidData()
+        {
+            var expextedResult = _fixture.Build<Product>().CreateMany().ToList();
```

Comment:
// arrange 
// act 
// assert
and always try and leave line breaks between them if possible.